### PR TITLE
Added GitVio to Open-Source

### PR DESCRIPTION
**Pull request by devXprite**
I have added GitVio to Open-Source Projects.

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
@devXprite please resolve the merge conflict

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: Open-Source/Contribute/Readme.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 - [facebook-chat-api](https://github.com/Schmavery/facebook-chat-api)
 - [flexy](https://github.com/vladocar/flexy)
 - [GFOSS - Open Technology Alliance](https://summerofcode.withgoogle.com/archive/2021/organizations/5185060013080576)
+- [GitVio](https://github.com/devXprite/GitVio/)
```
Comment (kaiwalyakoparkar) [inline Open-Source/Contribute/Readme.md:25]: LGTM, thank you so much for contributing

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kaiwalyakoparkar — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by devXprite** [comment]
Thanks @kaiwalyakoparkar for approving.

**Pull request merged**